index.js

```diff
@@ -130,8 +131,6 @@ class KeyringController extends EventEmitter {
     if (!firstAccount) {
       throw new Error('KeyringController - First Account not found.');
     }
-
-    await this.persistAllKeyrings(password);
```

Comment:
Removed because the last time `this.keyrings` is modified is within `addNewKeyring`, which calls `persistAllKeyrings` internally.